src/components/pages/dashboard-pages/EditLesson.jsx

```diff
@@ -0,0 +1,110 @@
+import React, { useState, useEffect } from 'react';
```

Comment:
Stretch goal: If you have time I would add a simple message that can be displayed to the user of required fields. e.g. if you try to edit without a title nothing happens. There is no indication of why the form is not being submitted.

Reply:
Thank you! Done)